console/v1/types_console_notification.go

```diff
@@ -0,0 +1,50 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Extension for configuring openshift web console notifications.
+type ConsoleNotification struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              ConsoleNotificationSpec `json:"spec"`
+}
+
+type ConsoleNotificationSpec struct {
+	// Text of the notification
+	Text string `json:"text"`
+	// The location of the notification (top, bottom, topBottom)
+	Location NotificationLocationSelector `json:"location"`
+	// Object that holds notification link details
```

Comment:
Optional?

Reply:
Yeah, I suppose no need to require a link. Ill annotate.